lib/Parser/RegexPattern.h

```diff
@@ -15,8 +15,17 @@ namespace UnifiedRegex
     class Matcher;
     struct TrigramInfo;
 
+    static const uint TestCacheSize = 8;
+
+    struct RegExpTestCache
+    {
+        Field(bool) result;
```

Comment:
nit: I see you just moved this definition from other file. Did you consider not creating this additional struct? the RegextPattern class below has bit pattern we can adjust field result there without any cost.

Reply:
testCache is an array, so can't just move result out. but there is a lot of wasted space for padding which i think can be improved

---
In reply to: [150975823](https://github.com/Microsoft/ChakraCore/pull/4207#discussion_r150975823) [](ancestors = 150975823)